console/console-init/ui/src/components/TypeAheadSelect/TypeAheadSelect.tsx

```diff
@@ -75,6 +75,7 @@ const TypeAheadSelect: React.FunctionComponent<ITypeAheadSelectProps> = ({
           } else {
             setOptions([
               <SelectOption
+                key={0}
```

Comment:
can we replace the key with dynamic value like index?

Reply:
There is no dynamic value here, as there is only one element.